de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/simulator/JSSimulatorNodeModel.java

```diff
@@ -250,81 +245,31 @@ private static JSSimulation toJSSimulation(FskSimulation fskSim, List<Parameter>
 
   @Override
   protected void saveSettingsTo(NodeSettingsWO settings) {
-
-    try {
-      final JSSimulatorViewValue vv = getViewValue();
-      saveJsonSetting(vv.simulations, vv.modelMath);
-    } catch (IOException | CanceledExecutionException e) {
-      e.printStackTrace();
-    }
+    m_config.saveSettings(settings);
   }
 
   @Override
   protected void loadValidatedSettingsFrom(NodeSettingsRO settings)
       throws InvalidSettingsException {
-    try {
-      loadJsonSetting();
-    } catch (IOException | CanceledExecutionException e) {
-      e.printStackTrace();
-    }
+    m_config.loadSettings(settings);
   }
 
-
-  protected void loadJsonSetting() throws IOException, CanceledExecutionException {
-    final File directory =
-        NodeContext.getContext().getWorkflowManager().getContext().getCurrentLocation();
-    final String containerName = buildContainerName();
-
-    final File settingFolder = new File(directory, containerName);
-   
-    // Read configuration strings
-    final String simulationString = NodeUtils.readConfigString(settingFolder, "simulations.json");
-
-    // Update view value
-    final JSSimulatorViewValue viewValue = getViewValue();
-
-    // Parse simulation string and update view Value
-    final Pair<Integer, List<JSSimulation>> simulationsInfo =
-        SimulationSetting.from(simulationString);
-    viewValue.selectedSimulationIndex = simulationsInfo.getFirst();
-    viewValue.simulations = simulationsInfo.getSecond();
-
-    viewValue.modelMath = NodeUtils.readConfigString(settingFolder, "modelMath.json");
+  private static String FromOjectToJSON(final Object object) throws JsonProcessingException {

```

Comment:
This method is only used once in getViewValue and MAPPER.writeValueAsString(object) could just be used there.